crates/web-plugins/didcomm-messaging/protocols/basic-message/src/model.rs

```diff
@@ -0,0 +1,21 @@
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct BasicMessage {
+    /// Message ID (unique identifier)
+    pub id: String,
+    /// Message type URI for basic messages
+    pub message_type: String,
+    /// Localization information for message language
+    pub locale: Option<String>,
```

Comment:
Why is local optional? I don't see this in the spec.

Reply:
Local is for lang

Reply:
> Local is for lang

I know. I'm asking why it is optional. Can you link the protocol spec that state it?